.gitignore

```diff
@@ -132,6 +132,10 @@ cppcheck-cppcheck-build-dir/
 *.pydevproject
 *.launch
 
+# Emacs
+\#*#
```

Comment:
Doesn't this need to be `\#*\#` for the second `#` to be accepted? Otherwise that's a comment.

Reply:
Oh, right!